Lib/test/test_free_threading/test_uuid.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import os
+
+from test.support import import_helper, threading_helper
+from test.support.threading_helper import run_concurrently
+
+c_uuid = import_helper.import_module("_uuid")
+
+NTHREADS = 10
+UUID_PER_THREAD = 1000
+
+
+@threading_helper.requires_working_threading()
+@unittest.skipUnless(c_uuid, "requires the C _uuid module")
```

Comment:
this isn't needed as import_module would automatically skip it if `_uuid` is not present.